Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.56-slim as builder
+ENV builddeps="cmake git clang"
+WORKDIR /usr/src/ion-cli
+COPY . .
+RUN apt-get update -y \
+  && apt-get install -y ${builddeps} \
+  && git submodule update --init --recursive
+RUN cargo install --path .
```

Comment:
It'd be nice to make sure this configuration works each time we commit new code. How easy would it be to add building this Docker image to the GitHub actions?

Reply:
Agreed, this should definitely have an action.